plugins/splunk-on-call/src/components/EntitySplunkOnCallCard.tsx

```diff
@@ -109,7 +109,12 @@ const useStyles = makeStyles({
   },
 });
 
-export const EntitySplunkOnCallCard = () => {
+type Props = {
+  readOnly?: boolean;
+};
+
+export const EntitySplunkOnCallCard = ({ readOnly }: Props) => {
```

Comment:
for ultimate cleanliness, could

- `export` the props above,
- rename the props to match component,
- unpack the props inside the function,
- mark both with `/** public */`

this will make for a ✨ neat ✨ api report (and future generated API docs)!

so in the end,

```ts
/** @public */
export const EntitySplunkOnCallCard = (props: EntitySplunkOnCallCardProps) => {
  const { readOnly } = props;
```

Reply:
@freben Good call! I believe I've made these changes.